todoapp/src/components/Todo.tsx

```diff
@@ -168,13 +172,113 @@ const useStyles = makeStyles({
         }
       }
     }
+  },
+  popUp: {
+    "&&": {
+      position: 'absolute',
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
+      backgroundColor: 'white',
+      width: '300px',
+      p: '1rem',
+      borderRadius: '8px',
+    }
+  },
+  popUpButton: {
+    "&&": {
+      width: '100%',
+      backgroundColor: '#ff5252',
+      color: 'white'
+    }
   }
 })
 
+type Todo = {
+  id: string;
+  text: string;
+  completed: boolean
+};
+
 function Todos(): JSX.Element {
   const isSmallScreen = useMediaQuery('(max-width:600px)');
   const classes = useStyles();
+  const [text, setText] = useState("");
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
```

Comment:
you are using useStates here, instead of this, use hookStates for state management as asked in the guidelines